### DISCUSSION TITLE
Nominatim with coolify

### DISCUSSION THREAD

**Discussion by viisrael** [coollabsio/coolify#3617] [Service Template Requests]
Hey, guys!

Did someone here configure Nominatim with Coolify? How was your experience? Do you recomend?

**Comment by alguintu** [comment]
Did you make it work? Banging my head for a couple of days now. No matter what strategy I try (peter-evans, mediagis, custom Dockerfiles or docker-compose) I can't seem to make it work. I either get bad gateways or no available servers.

**Reply by viisrael** [reply]
I ran it in my local machine, created a image from this container and sent it to my docker registry.

In my coolify, I configured it to use this image

**Comment by breadwithmeth** [comment]
Hello don't know if you still need It. You just need to wait about 30 minutes. Don't forget to set env value PBF_URL
